client/test/pages/snipetts-index.test.js

```diff
@@ -1,30 +0,0 @@
-// /*eslint-disable no-unused-vars*/
```

Comment:
for now I've skipped this cause I've made test for composing components. We have to think if page tests are needed.